README.md

```diff
@@ -9,14 +9,10 @@ To start up a local sandbox environment of the Northlight UI framework:
 1. Clone this repository
 2. cd into the directory
 3. run `yarn`
-4. cd into `/tokens`
-5. run `yarn build`
-6. cd into `/icons`
-7. run `yarn build`
-8. cd into `/framework`
-9. run `yarn start`
-
-This will spin up a local environment sandbox at localhost:3008
+4. cd into `/framework`
+5. run `yarn nx demo`
```

Comment:
it's enough with yarn demo right?

Reply:
it is not

Reply:
nx runs all dependant builds that is needed since you did the last build

Reply:
You can alternatively install nx globally and run `nx demo`